src/transforms/pull-request.ts

```diff
@@ -82,7 +82,7 @@ export default async (github: GitHubAPI, pullRequest: Octokit.PullsGetResponse,
 				? []
 				: [
 					{
-						createPullRequestUrl: generateCreatePullRequestUrl(pullRequest?.head?.repo?.html_url, pullRequest?.head?.ref, issueKeys), 
+						createPullRequestUrl: generateCreatePullRequestUrl(pullRequest?.head?.repo?.html_url, pullRequest?.head?.ref, issueKeys || undefined), 
```

Comment:
I wonder if we should explicitly do issueKeys || undefined ? 

Looks like an excessive check. (Otherwise we'd better move it inside the function)